src/main.ts

```diff
@@ -32,17 +33,31 @@ import { isNotFoundError } from './utils/http'
 
 const app = createApp(App)
 const pinia = createPinia()
-const i18n = setupI18n()
 const head = createHead()
 
 routerPromise
   .then((router) => {
     app.use(router)
     app.use(pinia)
-    app.use(i18n)
     app.use(head)
     app.use(TextClamp)
     app.use(LoadingPlugin)
+    app.use(datagouv, {
+      name: 'data.gouv.fr',
+      baseUrl: config.datagouvfr.base_url,
+      apiBase: config.datagouvfr.base_url,
+      staticUrl: 'https://static.data.gouv.fr',
```

Comment:
This needs to change when targeting demo, right? If so, we need a new setting for that.

Reply:
Works pretty well with current configuration file too :p